mesheryctl/internal/cli/root/system/export.go

```diff
@@ -0,0 +1,201 @@
+package system
```

Comment:
@Althaf66 This file should be placed in the model package. If you're organizing model commands into separate files, please ensure they are all located within a single package, as demonstrated for [component command](https://github.com/meshery/meshery/tree/master/mesheryctl/internal/cli/root/components).

Reply:
@Althaf66 thoughts?